study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/ActivityBuilder.java

```diff
@@ -318,11 +318,24 @@ public Config readDefinitionConfig(String filepath) {
             throw new DDPException("Activity definition file is missing: " + file);
         }
 
+        Config definition = ConfigFactory.parseFile(file).resolveWith(varsCfg);
```

Comment:
Wait, to keep existing behavior, `readDefinitionConfig(String filepath)` should call the other method and pass in true. Something like this:
```java
public Config readDefinitionConfig(String filepath) {
  return readDefinitionConfig(filepath, true);
}

public Config readDefinitionConfig(String filepath, boolean allowUnresolved) {
  ...
  if allowUnresolved {
    definition = ...resolveWith(varsCfg, ...);
  } else {
    definition = ...resolveWith(varsCfg);
  }
  ...
}
```

Reply:
And then we should update `UpdateTemplatesInPlace` to call `readDefinitionConfig(filepath, false)`.